### DISCUSSION TITLE
Ragel under Windows?

### DISCUSSION THREAD

**Discussion by mmaypo** [adrian-thurston/ragel#93] [General]
Hi - wanting to run Intel Hyperscan on a Windows 10 box, and ragel is a dependency. Has anyone got a cmake, or something windows native to build ragel?
thx

**Comment by ayruskrish** [comment]
Hi I am also trying to do the same is there any way to compile?

**Comment by PolarGoose** [comment]
@mmaypo and @ayruskrish,
I have created a repository to build Ragel on Windows: https://github.com/PolarGoose/Ragel-for-Windows
You can download the Windows binary from there.
